src/webui/src/components/TrialsDetail.tsx

```diff
@@ -336,12 +337,24 @@ class TrialsDetail extends React.Component<TrialsDetailProps, TrialDetailState>
                     let expLogCollection: boolean = false;
                     const isMultiy: boolean = res.data.params.multiPhase !== undefined
                         ? res.data.params.multiPhase : false;
+                    const tuner = res.data.params.tuner;
+                    // I'll set optimize is maximize if user not set optimize
+                    let optimize: string = 'maximize';
+                    if (tuner !== undefined) {
+                        if (tuner.classArgs !== undefined) {
+                            if (tuner.classArgs.optimize_mode !== undefined) {
+                                if (tuner.classArgs.optimize_mode !== 'maximize') {
```

Comment:
If we default to maximize, this should be `optimize_mode === 'minimize'`

Reply:
So, if user set this filed, it's value must be minimize or maximize.
Ok. I'll change my code.

Reply:
I mean, it can be something else, but in that case we should choose the default behavior.

Reply:
Got it, thanks very much.